vision/google/cloud/vision/_http.py

```diff
@@ -48,8 +49,9 @@ def annotate(self, image, features):
         data = {'requests': [request]}
         api_response = self._connection.api_request(
             method='POST', path='/images:annotate', data=data)
-        responses = api_response.get('responses')
-        return responses[0]
+        images = api_response.get('responses')
+        if len(images) == 1:
+            return Annotations.from_api_repr(images[0])
```

Comment:
Also, once I get this GAPIC support added, I have two other PRs. One for proper handling of batch images and another for handling errors received from the service.